var/spack/repos/builtin/packages/ncbi-toolkit/package.py

```diff
@@ -34,6 +41,8 @@ class NcbiToolkit(AutotoolsPackage):
 
     depends_on("boost@1.35.0:+test+log")
     depends_on("bzip2")
+    depends_on("cpio", type="build")
+    depends_on("diffutils", type="build")
```

Comment:
Are these for a specific version on (so could use a `when=` arg) or do they apply to all versions?

Reply:
I believe they apply to all versions. The container I was building it for rhel9 didn't have cmp or cpio in it, so I was hitting these.